scripts/lib/CIME/test_scheduler.py

```diff
@@ -218,13 +218,15 @@ def __init__(self, test_names, test_data=None,
                         else:
                             self._update_test_status(test, phase, TEST_PEND_STATUS)
                             self._update_test_status(test, phase, status)
+                logger.info("Using existing test directory %s"%(self._get_test_dir(test)))
         else:
             # None of the test directories should already exist.
             for test in self._tests:
                 expect(not os.path.exists(self._get_test_dir(test)),
                        "Cannot create new case in directory '%s', it already exists."
                        " Pick a different test-id" % self._get_test_dir(test))
-        logger.info("Created test in directory %s"%self._get_test_dir(test))
+                logger.info("Creating test directory %s"%(self._get_test_dir(test)))
+
```

Comment:
As above, since you moved the statement inside the loop, please add the test name to the log.